db/migrations/20240802044339_player_use_user_id_over_username.go

```diff
@@ -0,0 +1,116 @@
+package migrations
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/pressly/goose/v3"
+)
+
+func init() {
+	goose.AddMigrationContext(upPlayerUseUserIdOverUsername, downPlayerUseUserIdOverUsername)
+}
+
+func upPlayerUseUserIdOverUsername(ctx context.Context, tx *sql.Tx) error {
+	// Additional note; we also rename ip_address to IP. For some bizarre
```

Comment:
I don't think this comment should be here, I think it is enough for it to be mentioned in the PR itself.